libs/sdk-ui-dashboard/src/model/tests/fixtures/SimpleDashboard.fixtures.ts

```diff
@@ -175,6 +276,40 @@ export const drillToToInsightFromWonMeasureDefinition: IDrillToInsight = {
     transition: "pop-up",
     target: SimpleSortedTableWidgetInsightRef,
 };
+
+export const drillToCustomUrlFromMeasureDefinition: IDrillToCustomUrl = {
```

Comment:
please keep the global constants capitalized.

I also suggest (not necessary now, perhaps quick followup) to categorize these mocks. The constants names are lenghty and the names are too generic. if we have something like:

SimpleDashboardDrillDefinitions: {
     CustomUrlFromMeasure: ...
     AttributeUrlFromMeasure: ...
}

then it will not clutter the space so much